src/components/Layout.tsx

```diff
@@ -14,28 +14,26 @@ type Props = {
 const Layout = ({ text, icon, children }: Props) => {
   return (
     <>
-      <Flex alignItems="center" h={36} bg="teal.500">
+      <Flex bgColor="teal.500" h={36} align="center">
         <HSpacer size={16} />
-        <HStack w="100%" h="100%">
-          <HStack w="90%" alignItems="start">
-            <VSpacer size={1} />
-            <Icon as={icon} boxSize={6} color="teal.600" />
-            <Text
-              fontSize="lg"
-              lineHeight={7}
-              fontWeight="semibold"
-              color="white"
-              whiteSpace="pre-line"
-            >
-              {text}
-            </Text>
-          </HStack>
+        <Flex>
```

Comment:
ヘッダー部を上下センタリングしておく必要性を感じないから、外側の `Flex` の配置指定(子)は デフォルトの `start` のままで、内側の `Flex` の配置指定(自身) を `center` でもいい気がする

```suggestion
        <Flex alignSelf="center">
```